user/models.py

```diff
@@ -10,6 +11,24 @@
 from trips.models import UserLocation
 
 
+def change_username(username):
+    try:
+        # We stop when the exception is caught
+        while True:
+            str_number = re.match('.*?([0-9]*)$', username).group(1)
+            if str_number == '':
+                username = username + '1'
+            else:
+                number = int(str_number)
+                number = number + 1
+                username = re.sub(r'\d+$', '', username)
```

Comment:
I don't extract it from the previous regex because the previous regex only match the last digit